manifests/mod/remoteip.pp

```diff
@@ -0,0 +1,27 @@
+class apache::mod::remoteip (
+  $header            = 'X-Forwarded-For',
+  $proxy_ips         = [ '127.0.0.1' ],
+  $proxies_header    = undef,
+  $trusted_proxy_ips = undef,
```

Comment:
this class has n configurable parameters parameters exposed, none of which are documented in the README.